js/about/adblock.js

```diff
@@ -80,17 +81,17 @@ class AboutAdBlock extends React.Component {
               ? <div className='adblockLastETag'><span data-l10n-id='lastCheckETagLabel' /> <span>{this.state.adblock.get('etag')}</span></div>
```

Comment:
we can remove  `adblockLastETag` and also `adblockLastChecked` class as well. np keeping div element.